### ISSUE TITLE
make homepage main padding tighter

### ISSUE THREAD

**Issue by theDavidBarton** [theDavidBarton/theDavidBarton.github.io#147]
14rem on both sides would look better

**Comment by theDavidBarton** [comment]
better to done with:
```css
main {
    max-width: 50rem;
    margin-left: auto;
    margin-right: auto;
}
```

can be done after full bootstrap to CSS grid migration 😉

**Comment by theDavidBarton** [comment]
closed by #152, implemented while dropping Bootstrap 4.
